librz/analysis/meta.c

```diff
@@ -5,6 +5,7 @@
 
 #include <rz_analysis.h>
 #include <rz_core.h>
+#include <rz_io.h>
```

Comment:
Forgotten?

Reply:
Yes i forgot it

Reply:
I accidentally deleted the `#include <rz_io.h>` of analysis.c, but there seems to be no problem